api/models/Centres.js

```diff
@@ -5,6 +5,14 @@ var ValidationError = require('../lib/exceptions/ValidationError');
 var BedCountService = require('../services/BedCountService');
 var LinkingModels = require('sails-linking-models');
 
+// helpers
+const unreconciledMovementReducer = (movements, gender, direction) => movements.reduce((reduced, m) => {
+  if (m.gender === gender && m.direction === direction) {
+    reduced.push({ id: m.id, cid_id: m.cid_id });
+  }
```

Comment:
So the `cid_id` is the number we want to display to the users, meanwhile the `id` is a unique identifier from the database which is required by the Angular `track by` statement to avoid problems where `cid_id`s might not be unique (e.g. the same person checking in, then out, then in again).